tools/node-modules

```diff
@@ -109,6 +109,29 @@ Next steps:
 EOF
 }
 
+cmd_verify() {
+    # Verifies that the package.json and node_modules of the given commit match.
+    commit="$(git rev-parse "$1:node_modules")"
+    fetch_sha_to_cache "${commit}"
+
+    committed_tree="$(git_cache rev-parse "${commit}^{tree}")"
+    expected_tree="$(git cat-file blob "$1:package.json" | cmd_npm_install | tar_to_cache)"
+
+    if [ "${committed_tree}" != "${expected_tree}" ]; then
+        exec >&2
+        printf "\nCommit %s package.json and node_modules aren't in sync!\n\n" "$1"
```

Comment:
What does this parameter mean? I naïvely ran
```
❱❱❱ tools/node-modules verify
tools/node-modules: line 122: $1: unbound variable
```
It's already referenced in line 114, but I suppose the `$()` subshell doesn't inherit `set -u`.

Running it with `HEAD` argument works fine. Would that be a sensible default?